cosmovisor/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/cosmos/cosmos-sdk/cosmovisor
 go 1.18
 
 require (
-	github.com/cosmos/cosmos-sdk v0.46.0
+	github.com/cosmos/cosmos-sdk v0.46.0-beta2.0.20220808215643-b6791b7af44b
```

Comment:
should be 0.46.0

Reply:
ah, I see.  This can cause a thing....

Will take me a moment to figure out.

Reply:
Yeah this is because you did `go work sync` after having added cosmovisor in the `go.work`

Reply:
@faddat, meaculpa, removing this was breaking `make test`. We have a few solutions to fix that:

- In `run-tests` (Makefile), as `GOWORK=off` before running `go test`. This requires (although I think it's good), packages to be properly tagged to pass CI.
- Accept that it will upgrade the tendermint version until the SDK is downgrading to 34.x on main as well (as the support of 0.35 is anyway dropped by TM)

Reply:
I am actually for doing both, merging with cosmovisor in the go.work but then still disable it in the CI runs to force us to tag properly our versions. Wdyt @faddat, @marbar3778?